src/Microsoft.Diagnostics.DebugServices.Implementation/Module.cs

```diff
@@ -216,8 +216,8 @@ public string GetSymbolFileName()
                        (ex is InvalidVirtualAddressException ||
                         ex is ArgumentOutOfRangeException ||
                         ex is IndexOutOfRangeException ||
+                        ex is OverflowException ||
```

Comment:
Did this OverflowException come from somewhere in ClrMD?  I've had accidental OverflowExceptions coming out in the past so I want to make sure I didn't introduce a new one.

Reply:
No, that OverflowException wouldn't come from CLRMD.  This code is calling some code in the symstore ELF reader that I copied. I don't remember exactly why it catches overflow.